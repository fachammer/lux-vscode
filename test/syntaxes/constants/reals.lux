1.0
.0
.123
0.123e10
-1.0
-.1
+1.0
-123e-20123