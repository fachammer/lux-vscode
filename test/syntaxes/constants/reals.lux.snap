>1.0
#^^^ source.lux constant.numeric.real.lux
>.0
#^^ source.lux constant.numeric.real.lux
>.123
#^^^^ source.lux constant.numeric.real.lux
>0.123e10
#^^^^^^^^ source.lux constant.numeric.real.lux
>-1.0
#^^^^ source.lux constant.numeric.real.lux
>-.1
#^^^ source.lux constant.numeric.real.lux
>+1.0
#^^^^ source.lux constant.numeric.real.lux
>-123e-20123
#^^^^^^^^^^^ source.lux constant.numeric.real.lux